sources/api/migration/migrations/v1.12.0/oci-defaults-setting/src/main.rs

```diff
@@ -0,0 +1,31 @@
+#![deny(rust_2018_idioms)]
+
+use migration_helpers::common_migrations;
+use migration_helpers::{migrate, Result};
+use std::process;
+
+/// We added new settings for configuring the default OCI runtime spec,
+/// `settings.oci-defaults`, which will initially contain
+/// `settings.oci-defaults.capabilities` and
+/// `settings.oci-defaults.resource-limits`
+fn run() -> Result<()> {
+    if cfg!(variant_runtime = "k8s") {
+        migrate(common_migrations::AddPrefixesMigration(vec![
+            "settings.oci-defaults",
+            "services.oci-defaults",
+            "configuration-files.oci-defaults",
+        ]))
+    } else {
+        Ok(())
+    }
```

Comment:
Nit: are you forced to handle the branch? If you are not, it will be sufficient to just return `Ok(())`

Reply:
The compiler complained about not including `else`